src/main.js

```diff
@@ -16,9 +16,26 @@
 
 import Vue from 'vue';
 import App from './App.vue';
+import VueRouter from 'vue-router';
+import BCHDExplorer from './components/BCHDExplorer.vue';
 
 Vue.config.productionTip = false;
 
+Vue.use(VueRouter)
+const router = new VueRouter({
+  mode: 'history',
+  routes: [
+    {name: 'home', path: '/', component: BCHDExplorer},
+    {name: 'mainnet/address', path: '/mainnet/address/:address', component: BCHDExplorer},
+    {name: 'testnet/address', path: '/testnet/address/:address', component: BCHDExplorer},
+    {name: 'mainnet/block', path: '/mainnet/block/:blockHash', component: BCHDExplorer},
```

Comment:
Can't we do something like `path: '/:network/address/:address`? Then we can cut the routes in half?

Reply:
Makes sense.  Thanks for the feedback.  This is my first time working in vue.

Reply:
It was my first forray into Vue as well!